### DISCUSSION TITLE
VS Code extension for yo teams?

### DISCUSSION THREAD

**Discussion by stephanbisser** [pnp/generator-teams#328] [Ideas]
Starting this as a discussion so that everyone can add comments and ideas.
Basically we were discussing if we should create a Visual Studio Code extension for yo teams...

**Comment by RickVanRousselt** [comment]
Cool idea. Thinking about the MVP, we should at least be able to generate a new project, and maybe make it debuggable. 
Would be cool if something like this would be able to upgrade your Teamsgenerator to the new version (because VSCode does that automatically).

**Reply by stephanbisser** [reply]
Yep I'd say generate a project, test locally (so run the gulp serve ngrok command) would be cool as a starting point...
